src/main/kotlin/ru/hse/dynamomock/service/AWSDynamoDBMockService.kt

```diff
@@ -273,18 +274,31 @@ class AWSDynamoDBMockService(private val storage: DataStorageLayer) {
             .build()
     }
 
-    fun loadCSV(fileName: String, tableName: String) {
-        val allowedTypes = ImportableType.values().map { e -> e.name }
-        val header = mutableListOf<Pair<String, String>>()
+    fun loadCSV(filePath: String, tableName: String) {
         val rows = mutableListOf<List<String>>()
         csvReader {
             delimiter = ';'
             quoteChar = '"'
-        }.open(fileName) {
-            readAllAsSequence().forEach { row ->
-                rows.add(row)
+        }.open(filePath) {
```

Comment:
It's better to make InputStream from file and then call InputStream version of loadCSV with it